.github/workflows/verify-release-linux.yaml

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+name: Verify Release - Linux
+
+on:
+  workflow_dispatch:
+    inputs:
+      packageName:
+        description: 'Name of consul release package (consul vs consul-enterprise)'
+        required: true
+        default: 'consul'
+        type: choice
+        options:
+        - consul
+        - consul-enterprise
+      version:
+        description: The x.y.z version (also need to specify applicable suffixes like +ent and -dev)'
+        required: true
+        type: string
+
+jobs:
+  verify-ubuntu-amd64:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+      - name: docker build with version
+        run: |
+          docker build \
+          --build-arg PACKAGE=${{ inputs.packageName }} \
+          --build-arg VERSION=${{ inputs.version }} \
+          --build-arg TARGETARCH=amd64 \
+          -f ./build-support/docker/Verify-Release-Ubuntu.dockerfile .
+
+  verify-debian-amd64:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+      - name: docker build with version
+        run: |
+          docker build \
+          --build-arg PACKAGE=${{ inputs.packageName }} \
+          --build-arg VERSION=${{ inputs.version }} \
+          --build-arg TARGETARCH=amd64 \
+          -f ./build-support/docker/Verify-Release-Debian.dockerfile .
+
+  # verify-debian-arm64:
```

Comment:
Assuming these guys are commented out because they would need to run on arm64 runners.

If that's the case, can we leave a comment here explaining why they're present but commented out just so people don't interpret it as an accident?

Reply:
removed these in 3de90e1a8ae357774d920fb3829a11e276174893